x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="X.Org Xft library"
+DESCRIPTION="The X.Org project provides an open source implementation of the \
+X Window System. \
+The development work is being done in conjunction with the freedesktop.org \
+community. \
+The X.Org Foundation is the educational non-profit corporation whose \
+Board serves this effort, and whose Members lead this work."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="1989-95 GROUPE BULL
+	1998 Arnaud LE HORS
+	1996 Lorens Younes"
+# TODO: check all the licenses in the COPYING file in the next century.
+LICENSE="MIT"
```

Comment:
Since it is a custom license, I'd call it libXft.
We have done this for a few other recipes:
* x11-libs/libxcomposite/licenses/libXcomposite
* x11-libs/libxcursor/licenses/libXcursor
* x11-libs/libxdamage/licenses/libXdamage
* x11-libs/libxfixes/licenses/libXfixes
* x11-libs/libxrender/licenses/libXrender

(Each of these files is a copy of the `COPYING` file from the tarball.)

Reply:
Fixed.